### DISCUSSION TITLE
Feature Request: Multi-line textField support

### DISCUSSION THREAD

**Discussion by garffo** [skiptools/skip#100] [Ideas]
Just started using Skip, very cool!
I was wondering if it’s possible to add multi-line textFields?

I’ve tried:
```
TextField(“”, text: “”, axis: .vertical) 
TextEditor(text: $text)
```
Which all unfortunately seem to be unsupported.

**Comment by marcprux** [comment]
I've implemented a simple `TextEditor` in skip-ui 0.5.13 (https://github.com/skiptools/skip-ui/commit/7b3f82d74d7e3e40fecb969c50db9f49f53f3dc3). You should be able to use it after selecting Xcode's `File`/`Packages`/`Update to Latest Package Versions`.

Let us know if it works for you. We might also be able to implement `TextField(…, axis: .vertical)` without too much difficulty. We also encourage you to look into contributing these improvements yourself – it's [easier than you might think](https://skip.tools/docs/contributing/).

**Reply by garffo** [reply]
TextEditor works for me, I must've had another issue last time I tried. Cheers!
